### DISCUSSION TITLE
Problem with DrawingWidget

### DISCUSSION THREAD

**Discussion by delaere** [cduck/drawSvg#78] [Q&A] [answered]
Hi,

While it was working perfectly a few months ago, I am now unable to use the drawingWidget in Jupyter. I guess that this is related to the update of python and/or Jupyter.

I now get the following error message:

> Failed to create view for 'DrawingView' from module 'drawingview' with model 'DOMWidgetModel' from module '@jupyter-widgets/base'
> TypeError: Class constructor O cannot be invoked without 'new'

Is there anything that I can do to fix my installation?

Thanks!

**Comment by cduck** [comment, marked as answer]
Edit: This issue was reported in #80 and is now fixed.

---

Thanks for reporting the issue.  Try using notebook or "classic notebook", not Jupyter lab.  This is a known issue with Jupyter lab (#17).

If this is not the issue, please reply with the Python and Jupyter versions.  You're browser version may be important too.
```bash
python --version
python -m pip freeze
```

**Reply by cduck** [reply]
And please open a new github issue including the version information.  Or reply to #17 if appropriate.
